src/prefect/cli/worker.py

```diff
@@ -154,7 +154,7 @@ async def start(
                 )
             )
 
-            started_event = await worker._emit_worker_started_event()
+            started_event = await worker._emit_event_worker_started()
 
-    await worker._emit_worker_stopped_event(started_event)
+    await worker._emit_event_worker_stopped(started_event)
```

Comment:
note: i standardized these private event method names by making them all follow this convention:

```
def _emit_event.*
```

Reply:
no, no i wont